lib/PurchaseCmd.pm

```diff
@@ -0,0 +1,17 @@
+#!/user/bin/perl
```

Comment:
Please see the approach to isolate the Command to the OrderService on the ReceptionCmd and receptioncmd.t